chapters/en/Unit 2 - Convolutional Neural Networks/yolo.mdx

```diff
@@ -0,0 +1,73 @@
+# YOLO
+
+## Evolution of YOLO
+
+So far, we have seen the basic characteristics of YOLO and how it allows it to perform highly accurate and fast predictions. YOLOv1 was released in 2015 and until today multiple versions have been released. YOLOv1 was groundbreaking in terms of accuracy and speed. However, it suffered from poor localization and detection of multiple objects in a small image area.
+
+### **YOLOv2 (2016)** 
+
+After a year of releasing the first version, YOLOv2[1] came out. The improvements focused on both accuracy and speed but also dealt with the localization problem. Many improvements were made (image1). First, YOLOv2 replaced YOLOv1’s backbone architecture with Darknet-19, which is a variant of the Darknet architecture. Darknet-19 is lighter than the previous version’s backbone and it consists of 19 convolutional layers followed by max-pooling layers. This led to YOLOv2 having the ability to capture more information. Also, it included batch normalization to all convolutional layers removed the dropout layers dealing with the overfitting problem, and increased the mAP. It also introduced anchor boxes, which added prior knowledge to the detected width and height of the detected boxes (specifically they used). Additionally, to deal with the poor localization YOLOv2 predicted the class and objects for every anchor box and as previously grid (now 13x13). So we have a maximum (for 5 anchor boxes) of 13x13x5 = 845 boxes.
```

Comment:
what will be the `image1` ?

Reply:
It was in the first draft that I was making. I did not use it eventually, so it is a mistake. Thanks